### PR TITLE
mgr/dashboard: Increase max-width of popovers

### PR DESCRIPTION
**Pull request by tspmelo**
Fixes: https://tracker.ceph.com/issues/42334

Signed-off-by: Tiago Melo <tmelo@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tspmelo** [comment]
jenkins retest this please

**Comment by tspmelo** [comment]
jenkins test make check

**Comment by tspmelo** [comment]
jenkins test make check arm64

File: src/pybind/mgr/dashboard/frontend/src/styles.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@ $font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, 'Noto Sans', sans-s
 
 $badge-font-size: 1rem;
 $form-feedback-font-size: 100%;
+$popover-max-width: 350px;
```
Comment (votdev) [inline src/pybind/mgr/dashboard/frontend/src/styles.scss]:
```suggestion
.popover-content.popover-body div {
  max-width: 350px;
  text-overflow: ellipsis;
  overflow: hidden;
}
```
screenshot

Comment (tspmelo) [reply src/pybind/mgr/dashboard/frontend/src/styles.scss]: how about simply breaking the line? I don't like the ideia of hiding some of the information that you typed
image

**Pull request merged**